pkg/common/telemetry/metrics.go

```diff
@@ -59,7 +60,12 @@ func NewMetrics(c *MetricsConfig) *MetricsImpl {
 	// error and the implementation is currently no-fail.
 	sinks := metrics.FanoutSink{inmemSink}
 	sinks = append(sinks, c.Sinks...)
-	m, _ := metrics.New(metrics.DefaultConfig(c.ServiceName), sinks)
+
+	conf := metrics.DefaultConfig(c.ServiceName)
+	conf.EnableHostname = true
```

Comment:
I think we should prefer adding hostname as a label and not as a part of the metric name... systems which don't support labels can apply logic in the sink itself to inject labels into the metric name, that way systems which _do_ support labels aren't hindered

Reply:
Done.